tools/create-waardelijsten/app/source_models.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+from pydantic import BaseModel
```

Comment:
These models are for the tools purpose, and are just the pydantic models representation of the json file